File: C/Pattern_5.c
#include<stdio.h>
void main()
{
	int n,i,j,k;
	printf("Enter height : ");
	scanf("%d",&n);
	printf("\n");
	for(i=1;i<=n/2;i++)
	{
		for(k=i;k<n/2;k++)
		{
			printf(" ");
		}
		for(j=1;j<=i;j++)
		{
			printf(" *");
		}
		printf("\n");
	}
	for(i=1;i<=n/2;i++)
	{
		for(k=1;k<i;k++)
		{
			printf(" ");
		}
		for(j=i;j<=n/2;j++)
		{
			printf(" *");
		}
		printf("\n");
	}
}

File: Java/Pattern5.java
import java.io.*;
public class Pattern5
{
    public static void main(String args[])throws IOException
    {
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        int h,i,j;
        System.out.print("Enter height:");
        h=Integer.parseInt(br.readLine());
        for(i=h;i>=1;i--)
        {
            for(j=i;j>=1;j--)
            {
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
}
